wasm-chain-integration/src/v1/trie/low_level.rs

```diff
@@ -15,22 +15,21 @@ use std::{
     io::{Read, Write},
     iter::once,
     num::NonZeroU16,
-    slice::Iter,
     sync::{Arc, RwLock, RwLockReadGuard, RwLockWriteGuard},
 };
 
-const INLINE_CAPACITY: usize = 8;
+const INLINE_CAPACITY: usize = 4;
 
-#[derive(Default, Debug)]
+#[derive(Default, Debug, Clone)]
 /// An inner node in the [PrefixMap]. The default instance produces an empty
 /// node with no values and no children.
 struct InnerNode {
     value:    Option<NonZeroU16>,
     /// Children ordered by increasing keys.
-    children: Vec<KeyIndexPair>,
+    children: Vec<KeyIndexPair<8>>,
```

Comment:
I don't subscribe to having a constant `EIGHT = 8` for this.

What could make sense is to have a type alias `Key8IndexPair` or something like it, but since this is used so little I don't see the gain.

Reply:
The `N` is documented for `Chunk` but I added a comment to te KeyIndexPair type as well